_dev/src/store/modules/product-feed/mutations.ts

```diff
@@ -190,6 +190,7 @@ export default {
     state.selectedProductCategories = payload;
   },
   [MutationsTypes.SET_SYNC_SCHEDULE](state: LocalState, payload: boolean) {
+    localStorage.setItem('productFeed-requestSynchronizationNow', `${payload}`);
```

Comment:
```suggestion
    localStorage.setItem('productFeed-requestSynchronizationNow', payload);
```

Reply:
I need cast when `payload` is type-hint in boolean. Typescript told me string instead of boolean